x-pack/plugins/observability_solution/apm/public/components/routing/templates/apm_main_template.tsx

```diff
@@ -152,10 +157,14 @@ export function ApmMainTemplate({
         rightSideItems,
         ...pageHeader,
         pageTitle: pageHeaderTitle,
-        children:
-          showServiceGroupsNav && selectedNavButton ? (
-            <ServiceGroupsButtonGroup selectedNavButton={selectedNavButton} />
-          ) : null,
+        children: (
+          <EuiFlexGroup direction="column">
```

Comment:
I don't know if this will actually happen but if none of the conditions below are met, there will be an empty div in the DOM.

Reply:
You mean the component we pass as children? That won't happen unless we change the template params we pass. We  pass showServiceGroupsNav in ServiceGroupTemplate which is used for the services